### ISSUE TITLE
Fetch a question from the site

### ISSUE THREAD

**Issue by matheussampaio** [matheussampaio/urionlinejudge-cli#4]
Would be nice to have a command to fetch some problem from the site and create a file with the description injected on sample file.

# Example:

`sample.cpp`
```cpp
// inject:problem

int main()
{
    // work here

    return 0;
}
```
```bash
$ urionlinejudge fetch --id 1149 --path path/to/save --sample sample.cpp
```

`path/to/save/1149.cpp`:
```cpp
/*
 * URL: https://www.urionlinejudge.com.br/judge/en/problems/view/1149
 * 
 * Title: Summing Consecutive Integers
 * 
 * Description: Write an algorithm to read a value A and a value N. Print the sum of 
 * N numbers from A (inclusive). While N is negative or ZERO, a new N (only N) 
 * must be read. All input values are in the same line.
 *
 * Input: The input contains only integer values, ​​can be positive or negative.
 *
 * Output: The output contains only an integer value.
 * 
 * Sample Input: 
 * 3 2
 *
 * Sample Output:
 * 7
 *
 */
int main()
{
    // work here.

    return 0;
}
```

**Comment by matheussampaio** [comment]
@newlucasfevi Wanna do this?

**Comment by matheussampaio** [comment]
closes with bffd84efffc0ee331d6238299df665d968d02868
